README.md

```diff
@@ -0,0 +1,107 @@
+# OpenMME
+## Table of Contents
+[Introduction](#introduction)
+
+[Builds and Installation Procedure](#build-and-installation-procedure)
+
+[Update Config](#update-config)
+
+[Known Issues](#known-issues)
+
+[Config Parameters](#config-parameters)
+
+## Introduction:
+OpenMME is a grounds up implementation of the Mobility management Entity EPC S1 front end to the Cell Tower (eNB). Its design is performance optimized for high speed mobility events over the S1-MME interface, while maintaining state coherent high transaction rate interactions over the S6a interface to the HSS and the S11 interface to the Serving Gateway Control (SGWC). The design allows maximum utilization of the transaction rate allowed by the S1-MME Non-Access Stratum (NAS) messages over SCTP, S6a DIAMETER Attribute Value Pairs (AVPs) over TCP and S11 GTPV2C messages over UDP protocols.
+
+## Build and Installation Procedure
+### Update config 
+Update following config files
+    {install_path}/openmme/src/mme-app/conf/mme.json
+    {install_path}/openmme/src/s1ap/conf/s1ap.json
+    {install_path}/openmme/src/s11/conf/s11.json
+    {install_path}/openmme/src/s6a/conf/*.json
+    update the Diameter Configuration File:
+        {install_path}/openmme/src/s6a/conf/s6a_fd.conf
+       
+- the following values shall be defined:
```

Comment:
Should this be captured in the table?

Reply:
No..for now let it be lile this. Will update it as soon as i understand bit more about these files. net iterations can have better versions of the doc.